lib/ansible/plugins/inventory/azure_rm.py

```diff
@@ -450,7 +450,7 @@ def __init__(self, vm_model, inventory_client, vmss=None):
         self.nics = []
 
         # Azure often doesn't provide a globally-unique filename, so use resource name + a chunk of ID hash
-        self.default_inventory_hostname = '{0}_{1}'.format(vm_model['name'], hashlib.sha1(vm_model['id']).hexdigest()[0:4])
+        self.default_inventory_hostname = '{0}_{1}'.format(vm_model['name'], hashlib.sha1(to_bytes(vm_model['id'])).hexdigest()[0:4])
```

Comment:
How about multylining it for readability?
```suggestion
        self.default_inventory_hostname = (
            '_'.join((
                vm_model['name'],
                hashlib.sha1(to_bytes(vm_model['id'])).hexdigest()[0:4],
            ))
        )
```